packages/tenant-process/test/utils.ts

```diff
@@ -70,9 +79,40 @@ export const getMockTenant = (): Tenant => ({
   mails: [],
 });
 
-export const getMockAuthData = (organizationId?: string): AuthData => ({
-  organizationId: organizationId || uuidv4(),
-  userId: uuidv4(),
+export const currentDate = new Date();
+
+export const getMockVerifiedBy = (): TenantVerifier => ({
+  id: generateId(),
+  verificationDate: currentDate,
+});
+
+export const getMockRevokedBy = (): TenantRevoker => ({
+  id: generateId(),
+  verificationDate: currentDate,
+  revocationDate: currentDate,
+});
+
+export const getMockVerifiedTenantAttribute = (): VerifiedTenantAttribute => ({
+  id: generateId(),
+  type: "verified",
+  assignmentTimestamp: new Date(),
+  verifiedBy: [getMockVerifiedBy()],
+  revokedBy: [getMockRevokedBy()],
+});
+
+export const getMockCertifiedTenantAttribute =
+  (): CertifiedTenantAttribute => ({
+    assignmentTimestamp: currentDate,
+    id: generateId(),
+    type: "certified",
+    revocationTimestamp: currentDate,
+  });
+
+export const getMockAuthData = (
+  organizationId?: string & BRAND<"TenantId">
```

Comment:
there is now a branded type for this
```suggestion
  organizationId?: TenantId
```